app/liquid_tags/blogcast_tag.rb

```diff
@@ -0,0 +1,41 @@
+class BlogcastTag < LiquidTagBase
+  def initialize(tag_name, id, tokens)
+    super
+    @id = parse_id(id)
+  end
+
+  def render(_context)
+    html = <<-HTML
+      <div class="ltag_blogcast"
+        style="position: relative;
```

Comment:
This is causing some strange issues in the preview and post:
![2019-02-27 17 01 58](https://user-images.githubusercontent.com/17884966/53526146-76e74080-3ab1-11e9-93a1-85ab54877040.gif)

It seems to look fine without the inline CSS. I think it's best to remove it unless you had a specific reason for it.

Reply:
Removed it